dags/taar_amodump.py

```diff
@@ -1,83 +1,122 @@
-from airflow import DAG
 from datetime import datetime, timedelta
 
-from operators.emr_spark_operator import EMRSparkOperator
+from airflow import DAG
+from airflow.contrib.hooks.aws_hook import AwsHook
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.operators.subdag_operator import SubDagOperator
+
+from operators.gcp_container_operator import GKEPodOperator  # noqa
+from utils.dataproc import moz_dataproc_pyspark_runner
+
+# Unsalted cluster name so subsequent runs fail if the cluster name exists
+cluster_name = "bq-load-gke-1"
+
+taarlite_cluster_name = "taarlite-guidguid"
 
-from utils.mozetl import mozetl_envvar
+# Defined in Airflow's UI -> Admin -> Connections
+gcp_conn_id = "google_cloud_derived_datasets_3"
+connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
+
+# Dataproc connection to GCP
+gcpdataproc_conn_id = "google_cloud_airflow_dataproc"
+
+
+aws_conn_id = "airflow_taar_rw_s3"
+aws_access_key, aws_secret_key, session = AwsHook(aws_conn_id).get_credentials()
 
 default_args = {
-    'owner': 'amiyaguchi@mozilla.com',
-    'depends_on_past': False,
-    'start_date': datetime(2018, 11, 26),
-    'email': ['telemetry-alerts@mozilla.com', 'amiyaguchi@mozilla.com'],
-    'email_on_failure': True,
-    'email_on_retry': True,
-    'retries': 3,
-    'retry_delay': timedelta(minutes=30),
+    "owner": "vng@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 10, 7),
+    "email": ["telemetry-alerts@mozilla.com", "amiyaguchi@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 3,
+    "retry_delay": timedelta(minutes=30),
 }
 
-dag = DAG('taar_amodump', default_args=default_args, schedule_interval='@daily')
+dag = DAG("taar_amodump", default_args=default_args, schedule_interval="@daily")
 
-amodump = EMRSparkOperator(
+amodump = GKEPodOperator(
     task_id="taar_amodump",
-    job_name="Dump AMO JSON blobs with oldest creation date",
-    execution_timeout=timedelta(hours=1),
-    instance_count=1,
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    name="taar-amodump",
+    namespace="default",
+    image="gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.1",
```

Comment:
Comment to link to external github repo here would be nice

Reply:
I've added comments to each of the jobs to point to the repo now